services/bots/src/discord/commands/home-assistant/my.ts

```diff
@@ -96,7 +96,7 @@ export class CommandHomeAssistantMy implements DiscordTransformedCommand<MyDto>
     await interaction.reply({
       embeds: [
         new EmbedBuilder({
-          title: redirectData.name,
+          title: `Open ${redirectData.name} in your Home Assistant`,
           description: redirectData.description,
```

Comment:
On the My Website we [prefix descriptions](https://github.com/home-assistant/my.home-assistant.io/blob/main/src/entrypoints/my-create-link.ts#L249) with the text "Open your Home Assistant instance and". I think that we should do that here too.
```suggestion
          title: redirectData.name,
          description: `Open your Home Assistant instance and ${redirectData.description}`,
```

Reply:
Good idea!